dart/lib/src/sentry_options.dart

```diff
@@ -334,6 +334,22 @@ class SentryOptions {
     _scopeObservers.add(scopeObserver);
   }
 
+  final List<Type> _ignoredExceptionsForType = [];
+
+  /// Ignored exception types.
+  List<Type> get ignoredExceptionsForType => _ignoredExceptionsForType;
+
+  /// Adds exception type to the list of ignored exceptions.
+  void addExceptionFilterForType(Type exceptionType) {
+    _ignoredExceptionsForType.add(exceptionType);
+  }
+
+  /// Check if [ignoredExceptionsForType] contains an exception.
+  bool containsIgnoredExceptionForType(dynamic exception) {
+    return exception != null &&
+        _ignoredExceptionsForType.contains(exception.runtimeType);
```

Comment:
how is the behaviour on flutter web here? 

I think the runtimetype there is minified? afaik

Reply:
Yes, it's minified, but always has the same value during runtime, so the filter works as expected.

Reply:
got it 👍